### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by WinteryFox**

### CODE REVIEW

File: src/main/kotlin/app/jibiki/persistence/DatabaseAccessor.kt [hunk 1]
```diff
@@ -119,19 +119,29 @@ FROM (SELECT json_build_object(
                      'id', source.id,
                      'language', source.lang,
                      'sentence', source.sentence,
+                     'audio_uri',
+                     (SELECT 'https://audio.tatoeba.org/sentences/' || source.lang || '/' || source.id || '.mp3'
```
Comment (vendethiel) [inline src/main/kotlin/app/jibiki/persistence/DatabaseAccessor.kt:123]: why are you so addicted to subselects? why not a simple case statement?

File: src/main/kotlin/app/jibiki/persistence/DatabaseAccessor.kt [hunk 2]
```diff
@@ -119,19 +119,29 @@ FROM (SELECT json_build_object(
...
+                      WHERE source.audio IS NOT NULL),
                      'translations', json_agg(
                              json_build_object(
                                      'id', translations.id,
                                      'language', translations.lang,
-                                     'sentence', translations.sentence
+                                     'sentence', translations.sentence,
+                                     'audio_uri', (SELECT 'https://audio.tatoeba.org/sentences/' || translations.lang ||
```
Comment (vendethiel) [inline src/main/kotlin/app/jibiki/persistence/DatabaseAccessor.kt:130]: same

File: src/main/kotlin/app/jibiki/persistence/DatabaseAccessor.kt [hunk 3]
```diff
@@ -119,19 +119,29 @@ FROM (SELECT json_build_object(
...
+                                                          '/' || translations.id || '.mp3'
+                                                   WHERE translations.audio IS NOT NULL)
                                  )
                          )
                  ) json
       FROM links
-               JOIN sentences source
+               JOIN (SELECT id, lang, sentences.sentence, a.sentence audio
```
Comment (vendethiel) [inline src/main/kotlin/app/jibiki/persistence/DatabaseAccessor.kt:137]: why are these two joins subselects themselves?!

File: src/main/kotlin/app/jibiki/persistence/DatabaseAccessor.kt (hunk 1 above)
Comment (WinteryFox) [reply src/main/kotlin/app/jibiki/persistence/DatabaseAccessor.kt:123]: oh yea I forgot those existed, thanks for reminding me

**Pull request merged**